usr/src/uts/common/io/drm/drm_sunmod.c

```diff
@@ -236,20 +265,30 @@ drm_gem_map_access(devmap_cookie_t dhp, void *pvt, offset_t offset, size_t len,
 	struct gem_map_list *seg;
 
 	obj = (struct drm_gem_object *)pvt;
-	if (obj == NULL) {
-		dev = NULL;	/* avoid "used uninitialized" warning */
-		goto next;
-	}
 
-	dev = obj->dev;
-	if (dev->driver->gem_fault != NULL)
-		dev->driver->gem_fault(obj);
+	/*
+	 * Only call the driver fault handler after gtt_map_kaddr
```

Comment:
The VM system calls this before the back end has setup resources.
I don't remember why, but it was obvious we should just pass through
the faults to devmap_load() in such cases.